src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -48,7 +56,19 @@ public async Task<IActionResult> SubmitApplication(string accountId, Guid applic
 
             await _applicationService.Confirm(accountId, applicationId, email, string.Join(" ", firstName, lastName));
 
-            return RedirectToAction("BankDetailsConfirmation", "BankDetails", new { accountId, applicationId });
+            var accountLegalEntityId = await _applicationService.GetApplicationLegalEntity(accountId, applicationId);
```

Comment:
If the LegalEntityId is on the IncentiveApplicationDto then why not just add the Vrf status (assuming that that comes from the same location).

Also we should always favour returning resources from our services and then get the the properties form the data returned.  So in the  GetApplicationLegalEntity method we would call "/accounts/<accountId>/applications/<appId>
and then get the property from the resource and return that.  Then the API isn't coupled to the UI by having RPC type methods.  This makes it more reusable too since if we need some data from the resource for something else we already have it available.